test_requirements.txt

```diff
@@ -3,7 +3,7 @@
 astroid==1.3.8  # Pinning to avoid backwards incompatibility issue with pylint/pylint-django
 coveralls
 coverage==3.7.1
-django_nose==1.4.1
+django_nose>=1.4.1
```

Comment:
Necessary because older versions of django_nose are not compatible with django 1.11.